code/modules/cm_aliens/structures/fruit.dm

```diff
@@ -312,35 +342,52 @@
 
 /obj/effect/alien/resin/fruit/speed/consume_effect(mob/living/carbon/xenomorph/recipient, do_consume = TRUE)
 	if(mature && recipient && !QDELETED(recipient))
-		to_chat(recipient, SPAN_XENONOTICE("The [name] invigorates us to move faster!"))
-		new /datum/effects/xeno_speed(recipient, ttl = speed_duration, set_speed_modifier = speed_buff_amount, set_modifier_source = XENO_FRUIT_SPEED, set_end_message = SPAN_XENONOTICE("We feel the effects of the [name] wane..."))
+		new /datum/effects/xeno_speed(recipient, ttl = speed_duration, set_speed_modifier = speed_buff_amount, set_modifier_source = XENO_FRUIT_SPEED, set_end_message = SPAN_XENOWARNING("We feel the effects of the [name] wane..."))
+		to_chat(recipient, SPAN_XENOBOLDNOTICE("The [name] invigorates us to move faster!"))
+		recipient.balloon_alert(recipient, "we feel invigorated to run faster!", text_color = "#5B248C", delay = 1 SECONDS)
 	if(do_consume)
 		finish_consume(recipient)
 
+/obj/effect/alien/resin/fruit/speed/get_examine_text(mob/user)
+	. = ..()
+	if(ishuman(user))
+		. += "A basketball sized resin growth. Its translucent insides radiate faint purple light. It appears to be loosely attached to the weeds below."
+	if(isxeno(user) || isobserver(user))
+		. += "We sense eating this fruit will increase our speed by [SPAN_NOTICE("0.4")]. The effect lasts for [SPAN_NOTICE("15")] seconds."
```

Comment:
they have no idea how quick is that, either use percentage or simular. "By around half" etc